pkg/runner/local_docker.go

```diff
@@ -343,6 +348,34 @@ func ensureRedisContainer(cli *client.Client, log *zap.SugaredLogger, controlNet
 	return container.ID, err
 }
 
+// ensureSidecarContainer ensures there's a testground-sidecar container started.
+func ensureSidecarContainer(cli *client.Client, log *zap.SugaredLogger, controlNetworkID string) (id string, err error) {
```

Comment:
We might need a `testground sidecar update` command to update the sidecar service on a running cluster.

Reply:
https://github.com/ipfs/testground/issues/171